File: Baekjoon/S5/C#/S5_달팽이는 올라가고 싶다.cs
https://www.acmicpc.net/problem/2869

using System;

namespace Baekjoon
{
    class Program
    {
        static void Main(string[] args)
        {
            int[] arr = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
            int a = arr[0], b = arr[1], v = arr[2];
            Console.WriteLine(((v - b) % (a - b) == 0) ? (v - b) / (a - b) : (v - b) / (a - b) + 1);
        }
    }
}

File: Baekjoon/S5/JAVA/S5_달팽이는 올라가고 싶다.java
//https://www.acmicpc.net/problem/2869

import java.io.*;
import java.util.*;

public class Main{
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String str = br.readLine();
        StringTokenizer st = new StringTokenizer(str);
        int a = Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());
        int v = Integer.parseInt(st.nextToken());

        int ans = (v-b - 1)/(a-b) + 1;
        
        System.out.println(ans);
    }
}